magpie/api/webhooks.py

```diff
@@ -284,14 +284,14 @@ def setup_webhooks(config_path, settings):
     """
 
     settings["webhooks"] = defaultdict(lambda: [])
-    webhooks_conf = settings["webhooks"]  # type: WebhookConfig
+    webhooks_conf = settings["webhooks"]  # type: WebhookSettings
```

Comment:
Be careful with this naming. In `send_webhook_request()` `webhook_config` is of type `WebhookConfigItem`, while here `webhooks_conf` is of type `WebhookSettings`.

To avoid confusion, maybe use `webhooks_settings_config` here, instead.

Reply:
This is why I added the typings. 
Different devs using interchangeably the names for config/setttings. 

But I agree name can be more explicit also.

Reply:
Yes of course the choice of config or settings refers to the same concept. 
What's important is the consistence, with the variable naming/type, wether it's called a config or a setting.